concepticondata/conceptlists.tsv

```diff
@@ -168,3 +168,5 @@ Wheeler-2015-102	Wheeler, W. C. and Whiteley, Peter M.	2015		102	basic	English	U
 Janhunan-2008-235	Janhunan, Juha and Peltomaa, Marja and Sandman, Erika and Dongzhou, Xiawu	2008		273	areal	English	Wutun		Janhunan2008	Janhunan2008	This list was used to document basic vocabulary of the Wutun language, an apparently Sinitic language spoken in the Amdo region of Tibet. The numbering of the word list is rather strange, since it is either inconsistent, or there is a system that is not directly understandable for the reader. The authors list general terms which we know from normal basic vocabulary lists, but also cultural variants, yet apart from that, they also list items which are not given a further translation (and which we therefore had to leave unmapped).	118-127	
 Dunn-2011-146	Dunn, Michael and Burenhult, Niclas and Kruspe, Nicole and Tufvesson, Sylvia and Becker, Neele 	2011		146	areal	English	Aslian	http://www.jbe-platform.com/content/journals/10.1075/dia.28.3.01dun	Dunn2011		This list was created by Niclas Burenhult and Nicole Kruspe in order to document the Aslian languages of the highlands of the Malay Peninsula. This is based on the Swadesh list, but also includes some relevant culture/technology words.	291-323	
 Daniels-2016-85	Daniels, Don	2016		85	basic,areal	English	Magi,Aisi	http://dx.doi.org/10.1353/ol.2016.0004	Daniels2016	Daniels2016	This is a derived version of [Swadesh's list of 100 items (1955)](:ref:Swadesh-1955-100) used to document two languages of New Guinea.	219-221	
+Miedema-1985-	Miedema, J. and Welling, F. I.	1985		40	basic,areal	English	New Guinea		Miedema1985	Miedema1985	This is a list of 40 items 'based on Voorhoeve 1975' 	29-52	
```

Comment:
Do we have the original list bo Voorhoeve? Sorry for being a pain in the neck, but can we retrieve this source, or can we reference it by providing a link? It would be good if people who read this could trace it back. It is possible [this source](http://glottolog.org/resource/reference/id/300404), so one might just add the glottolog id, and all is fine.